packages/unit-testing/stubs/workbook/workbook.js

```diff
@@ -0,0 +1,927 @@
+define([
+	'./Aspect',
+	'./CalculatedMeasure',
+	'./Category',
+	'./Chart',
+	'./ChartAxis',
+	'./ChildNodesSelector',
+	'./Color',
+	'./ConditionalFilter',
+	'./ConditionalFormat',
+	'./ConditionalFormatRule',
+	'./Currency',
+	'./DataDimension',
+	'./DataDimensionItem',
+	'./DataMeasure',
+	'./Dataset',
+	'./DatasetLink',
+	'./DescendantOrSelfNodesSelector',
+	'./DimensionSelector',
+	'./Duration',
+	'./FieldContext',
+	'./FontSize',
+	'./Legend',
+	'./LimitingFilter',
+	'./MeasureSort',
+	'./MeasureValueSelector',
+	'./PathSelector',
+	'./Pivot',
+	'./PivotAxis',
+	'./PositionPercent',
+	'./PositionUnits',
+	'./PositionValues',
+	'./Range',
+	'./RecordKey',
+	'./ReportStyle',
+	'./Section',
+	'./Series',
+	'./Sort',
+	'./SortByDataDimensionItem',
+	'./SortDefinition',
+	'./TableColumn',
+	'./TableColumnCondition',
+	'./TableColumnFilter',
+	'./WorkbookInstance',
+], function () {
```

Comment:
All the elements defined above should be parameters of this function. They also need to be defined in the SuiteCloudJestConfiguration.js as modules.